dts/arm/nordic/nrf52840.dtsi

```diff
@@ -1,3 +1,4 @@
+#include <mem.h>
```

Comment:
nack, you shouldn't be modifying nrf52840.dtsi, probably need to include nrf52840_qiaa.dtsi instead.

Reply:
hmm, that does not work

Reply:
> hmm, that does not work

What happens?

Reply:
ok